src/app/notes/notes.effects.ts

```diff
@@ -23,7 +23,12 @@ export class NotesEffects {
     map((action: GetNotes) => action.filter),
     exhaustMap(filter =>
       this.notesService.getNotes(filter).pipe(
-        map((notes: Note[]) => {
+        map((noteList: NoteList) => {
+          const notes: Note[] = [];
+          for (const k of noteList.keys()) {
+            console.log('keys', k);
```

Comment:
Do we need this log message? If so I would prefer to use the logger implementation:
```suggestion
            this.logger.debug('keys', k);
```

Reply:
This was actually an artifact during debugging, I'll remove it.